test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoTest.java

```diff
@@ -1,456 +0,0 @@
-        
-        // Get the first page of 10 records
-        ForwardCursorPagedResourceList<ScheduledActivity> history = activityDao.getActivityHistoryV2(
-                healthCode, activityGuid, startDateTime, endDateTime, null, 10);
-        assertEquals(10, history.getItems().size());
-        history.getItems().stream().forEach(scheduledActivity -> assertEquals(MSK, scheduledActivity.getTimeZone()));
-        
-        Set<String> allTaskGuids = history.getItems().stream().map(ScheduledActivity::getGuid).collect(toSet());
-
-        // Get second page of records
-        String nextPageOffsetKey = history.getNextPageOffsetKey();
-        history = activityDao.getActivityHistoryV2(
-                healthCode, activityGuid, startDateTime, endDateTime, nextPageOffsetKey, 10);
-        assertEquals(10, history.getItems().size());
-        
-        // Assert request params are all set (nextPageOffsetKey is now the offsetKey submitted in the request) 
-        assertNotEquals(nextPageOffsetKey, history.getNextPageOffsetKey());
-        assertEquals(nextPageOffsetKey, history.getRequestParams().get("offsetKey"));
-        assertEquals(10, history.getRequestParams().get("pageSize"));
-        assertEquals(startDateTime.toString(), history.getRequestParams().get("scheduledOnStart"));
-        assertEquals(endDateTime.toString(), history.getRequestParams().get("scheduledOnEnd"));
-
-        assertNotEquals(nextPageOffsetKey, history.getOffsetKey());
-        assertEquals((Integer)10, history.getPageSize());
-        assertTrue(startDateTime.isEqual(history.getScheduledOnStart()));
-        assertTrue(endDateTime.isEqual(history.getScheduledOnEnd()));
-        
-        history.getItems().stream().forEach(scheduledActivity -> assertEquals(MSK, scheduledActivity.getTimeZone()));
-
-        // Now add the GUIDS of the next ten records to the set
-        allTaskGuids.addAll(history.getItems().stream().map(ScheduledActivity::getGuid).collect(toSet()));
-
-        // Should be 20 items in the set (that is, two entirely separate pages of GUIDs)
-        assertEquals(20, allTaskGuids.size());
-        
-        // Query for a time range that will produce no records
-        history = activityDao.getActivityHistoryV2(
-                healthCode, activityGuid, startDateTime, startDateTime, null, 10);
-        assertEquals(0, history.getItems().size());
-        assertNull(history.getNextPageOffsetKey());
-    }
-    
-    // Paging for this method can hit a situation where there are items with identical referentGuids. These 
-    // must page correctly, so this tests really pathological behavior where such runs of the same GUID 
-    // extend across multiple pages. You'd have to schedule the exact same task at the exact same time 
-    // over a hundred times to defeat the logic that pages this correctly.
-    @Test
-    public void getScheduledActivityHistoryV3PathologicalPaging() throws Exception {
```

Comment:
Similarly, should this also be moved to integ tests, with paging?

Reply:
See above